single_kernel_mongo/managers/cluster.py

```diff
@@ -115,7 +113,7 @@ def update_keyfile_and_hosts_on_mongos(self, relation: Relation) -> None:
 
         self.share_secret_to_mongos(relation)
 
-    def on_relation_broken(self, relation: Relation) -> None:
+    def cleanup_users(self, relation: Relation) -> None:
```

Comment:
love this change! can you make sure this same approach is taken in the other manager PRs?

Reply:
Yes I'll do it !